cluster/playbook/kubernetes.yaml

```diff
@@ -13,21 +13,32 @@
   roles:
     - {role: kubernetes/common}
 
-- name: Configure kubernetes master
-  hosts: master
+- name: Configure kubernetes primary master
+  hosts: master[0]
   remote_user: ubuntu
   become: yes
   environment:
     K8S_AUTH_KUBECONFIG: "{{kubeadmin_config}}"
   roles:
-    - {role: kubernetes/master}
+    - {role: kubernetes/master/common}
+    - {role: kubernetes/master/primary}
     - {role: kubernetes/addons/cni}
     - {role: kubernetes/addons/ingress, when: addons.ingress.enabled }
     - {role: kubernetes/addons/csi, when: provider == "digitalocean" and addons.csi.enabled }
     - {role: kubernetes/addons/ccm, when: provider == "digitalocean" and addons.ccm.enabled }
     - {role: kubernetes/addons/external_dns, when: provider == "digitalocean" and addons.external_dns.enabled }
     - {role: kubernetes/addons/cert_manager, when: provider == "digitalocean" and addons.cert_manager.enabled }
 
+- name: Configure kubernetes secondary master(s)
+  hosts: master[1:]
```

Comment:
@jskswamy Thanks for the comments. I have adopted like below:

Using `master[1:]` to run the role master/common, master/secondary
Using `master[0]` to run the role master/common, master/primary

I have tested with this new roles